sdk/python/tests/integration/scaffolding/test_repo_config.py

```diff
@@ -34,6 +34,37 @@ def _test_config(config_text, expect_error: Optional[str]):
         return rc
 
 
+def test_nullable_online_store_aws():
+    _test_config(
+        dedent(
+            """
+        project: foo
+        registry: "registry.db"
+        provider: aws
+        online_store:
```

Comment:
Aren't we agree on `online_store: null` format? why it's `type: null`?

Reply:
I misunderstood. Changed it to online_store: null